ui/src/app/views/workspace/component.ts

```diff
@@ -20,35 +20,35 @@ export class WorkspaceComponent implements OnInit {
 
   repositories: Repository[] = [];
   user: User;  // to detect if logged in
-  currentUrl: string;
-  cohortList = [];
+  cohortList: Cohort[] = [];
   constructor(
       private router: Router,
+      private route: ActivatedRoute,
       private userService: UserService,
       private repositoryService: RepositoryService,
       private cohortsService: CohortsService
   ) {}
-
+  // TODO: Replace arguments to getCohortsInWorkspace to read from URL.
   ngOnInit(): void {
     this.userService.getLoggedInUser().then(user => this.user = user);
     this.cohortsService
         .getCohortsInWorkspace(
-            WorkspaceComponent.DEFAULT_WORKSPACE_NS, WorkspaceComponent.DEFAULT_WORKSPACE_ID)
+            this.route.snapshot.url[1].path, this.route.snapshot.url[2].path)
```

Comment:
Maybe as a next task, if not as part of this PR, could you look into parsing/building URLs in a more semantic way? If the Router module can't solve it for us, we could at least put static build/parse functions on the components that own the URLs.

Just skimming, http://onehungrymind.com/named-router-outlets-in-angular-2/ looks like there's a way to subscribe to route.params and get the named path portions.